packages/vite-plugin-cloudflare/src/runner/module-runner.ts

```diff
@@ -18,10 +18,8 @@ export async function createModuleRunner(
 		throw new Error('Runner already initialized');
 	}
 
-	// we store the custom import file path in a variable to skip esbuild's import resolution
-	const workerdReqImport = '/__workerd-custom-import.cjs';
 	const { default: workerdCustomImport } = await (import(
-		workerdReqImport
+		`/${WORKERD_CUSTOM_IMPORT_PATH}`
```

Comment:
Is it OK to assume the `/` prefix here?

Reply:
I think so, because we set it at the root in the miniflare constructor, and within miniflare we specify the path just with `/` (without the need to add the windows drive letter)

Reply:
Can you see if it works just using `WORKERD_CUSTOM_IMPORT_PATH` directly without adding `/` in front? That's what they do in vitest-pool-workers.

Reply:
not including the `/` does seem to break things:
![Screenshot 2024-10-10 at 15 13 57](https://github.com/user-attachments/assets/57c99de0-ddd8-4e04-8515-626c799fc836)